### PR TITLE
Chore - General Cleanup

### PR DESCRIPTION
**Pull request by quentin41500**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,6 @@
+#Wed May 16 12:23:23 EDT 2018
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.7-all.zip
```
Comment (barnhill) [inline gradle/wrapper/gradle-wrapper.properties]: unless you need the sources for debugging Id suggest just using the binaries as its smaller and could decrease build times by limiting how much is downloaded

File: library/src/main/java/com/prolificinteractive/materialcalendarview/format/WeekDayFormatter.java [hunk 2]
```diff
@@ -17,5 +17,5 @@
     /**
      * Default implementation used by {@linkplain com.prolificinteractive.materialcalendarview.MaterialCalendarView}
      */
-    public static final WeekDayFormatter DEFAULT = new CalendarWeekDayFormatter(CalendarUtils.getInstance());
+    WeekDayFormatter DEFAULT = new CalendarWeekDayFormatter(CalendarUtils.getInstance());
```
Comment (tevjef) [inline library/src/main/java/com/prolificinteractive/materialcalendarview/format/WeekDayFormatter.java]: Missing another one in `com/prolificinteractive/materialcalendarview/format/DayFormatter.java`

Comment (quentin41500) [reply library/src/main/java/com/prolificinteractive/materialcalendarview/format/WeekDayFormatter.java]: Done

**Comment by tevjef** [comment]
LGTM 👍

**Pull request merged**